### PR TITLE
Fix remoting/etcdv3/client test cases

### PR DESCRIPTION
**Pull request by zjhmale**
Add up more assertations for the etcdv3 client.

### CODE REVIEW

**Comment by zouyx** [comment]
should make travis succes @zjhmale

**Comment by zjhmale** [comment]
@zouyx pls review again :)

File: remoting/etcdv3/client_test.go [hunk 1]
```diff
@@ -295,13 +299,26 @@ func (suite *ClientTestSuite) TestClientWatch() {
...
 			for _, event := range e.Events {
+				events = append(events, (*mvccpb.Event)(event))
 				t.Logf("type IsCreate %v k %s v %s", event.IsCreate(), event.Kv.Key, event.Kv.Value)
 			}
 		}
 
+		e1 := events[0]
+		e2 := events[1]
+
+		if e1.Type != mvccpb.PUT || string(e1.Kv.Key) != "name" || string(e1.Kv.Value) != "scott.wang" {
```
Comment (zouyx) [inline remoting/etcdv3/client_test.go]:
```suggestion
		assert.Contains()
```
if array is not always sorting, suggest use assert.Contains()

File: remoting/etcdv3/client_test.go [hunk 2]
```diff
@@ -334,25 +351,38 @@ func (suite *ClientTestSuite) TestClientRegisterTemp() {
...
-					return
+					break
 				}
 				t.Logf("type IsCreate %v k %s v %s", event.IsCreate(), event.Kv.Key, event.Kv.Value)
 			}
 		}
+
+		e1 := events[0]
+		e2 := events[1]
+		if e1.Type != mvccpb.PUT || string(e1.Kv.Key) != "scott/wang" || e1.Kv.Value != nil {
```
Comment (zouyx) [inline remoting/etcdv3/client_test.go]: as above

File: remoting/etcdv3/client_test.go (hunk 1 above)
Comment (zjhmale) [reply remoting/etcdv3/client_test.go]: thanks for the hint, but for the sake of event queue semantic, the watching channel should be a sequential generator which means the event created in the first place should always be the first one consumed outside the watching channel IIRC, maybe I'm wrong.

**Pull request merged**